README.md

```diff
@@ -527,7 +527,25 @@ let api = new ParseServer({
 
 ### Notes <!-- omit in toc -->
 
-- This feature is currently only available for MongoDB and not for Postgres.
+- There are no additional steps needed to use this feature with MongoDB
+- For Postgres, you will need to create a cron job, use [pgAdmin](https://www.pgadmin.org/docs/pgadmin4/development/pgagent_jobs.html), or similar to call the Postgres function, `idempodency_delete_old_rows()`. For example, the cron job can call the script below (make sure the script has the same priveledges to log into Postgres as your Parse Server):
```

Comment:
could you reword to:

> - For MongoDB, no additional steps are needed.
> - For Postgres, you need to create a cron job using [pgAdmin](https://www.pgadmin.org/docs/pgadmin4/development/pgagent_jobs.html) or similar to call a Postgres function `idempodency_delete_expired_records()` that deletes expired idempotency records. You can find an example script below. Make sure the script has the same privileges to log into Postgres as Parse Server.

i think "expired_records" is more universal than "old_rows". in case a future no-sql DB adapter is added and also don't support TTL, "row" may not be an accurate term and we'd end up with different functions names for different DB adapters.

Reply:
Updated